.github/workflows/lint.yml

```diff
@@ -165,12 +165,8 @@ jobs:
 
       - name: dependency by homebrew
         run: |
-          # Reopen for runner image v20231029.1
-          # old homebrew/core cause brew link and unlink not working
-          brew update  # No update because it is slow
-          # Force using python 3.11 from homebrew
-          brew unlink python
-          brew link --force --overwrite python@3.11
+          export HOMEBREW_NO_AUTO_UPDATE=1
```

Comment:
Wow love it.  This know-how needs to be kept among devs!

Reply:
yeah, I found that brew will do update automatically lol